demo-client/src/main/java/com/illia/client/model/holder/IMDbMovieHolderImpl.java

```diff
@@ -0,0 +1,52 @@
+package com.illia.client.model.holder;
+
+import com.illia.client.model.IMDbMovieEntity;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+@Component
+public class IMDbMovieHolderImpl implements IMDbMovieHolder {
+
+    private String fileName;
+    private List<IMDbMovieEntity> entities;
+
+    @Override
+    public String getFileName() {
+        return fileName;
+    }
+
+    @Override
+    public List<IMDbMovieEntity> getEntities(String fileName) {
+        if(!holdsFile(fileName) || entities.isEmpty()){
```

Comment:
It is a possible case that there are no entries in the file, therefore no entities, therefore not an exception case